### PR TITLE
Add TTWG

### PR DESCRIPTION
**Pull request by nigelmegitt**

### CODE REVIEW

**Comment by nigelmegitt** [comment]
@himorin @gkatsev please could you take a look at this? I don't have permissions on the onboarding repository to allow me to request a review from you.

File: template/34314 [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+implementation reports, registries, and ideas. This can be found at:
+  https://www.w3.org/wiki/TimedText
+
+https://w3c.github.io/Guide/participant/group.html?gid={{ group.id }}
+is a complete map of useful resources for new participants in this group.
+{{ group._links.homepage.href }} is also a useful resource.
+
+Should you have any question on how to get back up to speed with the
+Working Group, please get in touch with the Staff Contact of the group:
+Atushi Shimono <atsushi@w3.org>
```
Comment (xfq) [inline template/34314]: (minor nitpick: "Atushi" should be "Atsushi" per Hepburn romanization.)

Comment (nigelmegitt) [reply template/34314]: Thanks so much for spotting this. I don't think it is minor to misspell someone's name. I've fixed this at 922d353.

**Review by gkatsev — APPROVED** [review]
Sounds great, thanks for writing it up @nigelmegitt.

File: template/34314 [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+DST to 1500 UTC, but the date of the switch is discussed before each
+change).
+
+The weekly teleconference meeting agenda is published by email to the
+public reflector and worked on as a GitHub issue. Upcoming meetings
+have issues labelled as Agenda and can be found at:
+  https://github.com/w3c/ttwg/issues?q=is%3Aopen+is%3Aissue+label%3Aagenda
+
+The group usually meets face-to-face once or twice a year, but given the
+current circumstances, it is not clear when we might next meet F2F.
```
Comment (himorin) [inline template/34314]: we might use this for years, and could forget to update (e.g. due to less frequently), how about to put what is the 'current' - like `given the current circumstances (COVID-19)` or something.

File: template/34314 [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+During meetings, the group uses IRC to scribe and manage the speaker queue.
+An IRC webpage is provided with the agenda; you may also find it helpful
+to use a dedicated IRC client.
+
+A timeline of specifications currently being worked on is available at:
+  https://www.w3.org/AudioVideo/TT/wip/TTWG-2019-spec-timeline.html
```
Comment (himorin) [inline template/34314]: For this, also let me propose to have one landing page (with redirect to the newest, or listing all timelines) in /TT/wip/ (will push PR to ttwg repository shortly), and replace with URL of it...

File: template/34314 (hunk 1 above)
Comment (himorin) [reply template/34314]: (actually, it's `si` in Kunrei-shiki romanization which is in ISO...)

**Comment by caribouW3** [comment]
@himorin W3C team contacts should be able to merge their PRs (feel free to merge this one when you're ready and update as frequently as you wish)

**Comment by himorin** [comment]
> @himorin W3C team contacts should be able to merge their PRs 

ah, yes. For this, I'd want to change ttwg page side also, so want to hear how chairs think (so will wait reply on these).

File: template/34314 (hunk 2 above)
Comment (nigelmegitt) [reply template/34314]: Ah, yes, just noticed this comment. Good idea! Will do.

Comment (nigelmegitt) [reply template/34314]: Done

**Comment by himorin** [comment]
checked all links working.

**Review by himorin — APPROVED** [review]

**Pull request merged**